rest/kadai-rest-spring/src/main/java/io/kadai/common/rest/ldap/LdapClient.java

```diff
@@ -907,6 +913,22 @@ private Set<String> getPermissionIdsFromContext(final DirContextOperations conte
     return permissions;
   }
 
+  private boolean permissionsAreEmpty() {
+    return getUserPermissionsAttribute().isEmpty() || getPermissionSearchFilterName().isEmpty();
```

Comment:
I guess we can get null pointers here, or am I wrong?

Reply:
I am not sure. I will add a null check to be sure